### ISSUE TITLE
lgsvl service can't work

### ISSUE THREAD

**Issue by yaoweixiao-ux** [lgsvl/simulator#2127]
when I open lgsvl simulator, UI always show connecting to https://wise.svlsimulator.com
connection to cloud time out, giving up
Does lgsvl can't work now?

**Comment by sixiaixixiya** [comment]
Yes, I've noticed that too. Is there any other way to build a scene offline？

**Comment by zhuzhouzhou** [comment]
sad ~~~~ need a offline version

**Comment by Fin-Del-Mundo** [comment]
maybe need to use another simulator like carla

**Comment by sixiaixixiya** [comment]
> maybe need to use another simulator like carla

Does Carla have the same Scenario Editor like LGSVL?

**Comment by Fin-Del-Mundo** [comment]
> 

svl base on unity, but carla base on unreal.

**Comment by sixiaixixiya** [comment]
> > 
> 
> svl base on unity, but carla base on unreal.

I know it. But my question is that does Carla have the function of creating scenario by just dragging the obstacles in the simulation map like LGSVL's Visual Editor.

**Comment by ghost** [comment]
Sad news :(

Does anyone any other simulator which supports Apollo ? I've tried some carla-apollo bridge but they lack traffic signs support. As for whether Carla has scenario editor , I doubt they have one, but do refer to their documentations.

**Comment by YuqiHuai** [comment]
I started a project https://github.com/YuqiHuai/SORA-SVL back then to be prepared for this day. This is a local SVL server, it is basically an attempt to implement WISE, a.k.a. an offline version. I am very lucky to have many others helping to integrate features. I didn't get a chance to focus more on this project afterward, but maybe it makes sense to focus on it now.

I am also aware of another attempt to remove dependency from WISE https://github.com/OSSDC/OSSDC-SIM

**Comment by yaoweixiao-ux** [comment]
perfect!!!!
